content/reference/endpoints/mid_term_rate_maps.md

```diff
@@ -17,7 +17,7 @@ Check out our guide: [Understanding Midterm Pricing](/guides/understanding-midte
   <div class="tab-pane active" id="rates_read" markdown="1">
 Name             | Type    | Read/Write | Description
 -----------------|---------|------------|------------
-id               | Integer | Read       | Mid Term Rate Map's id.
+id               | String  | Read       | Mid Term Rate Map's id.
```

Comment:
would be more specific and say it's `uuid`,this can really help with types

Reply:
I set it as `BigInteger`, I noticed that this is on LOS records and other resources where id is uuid